src/test/http-mocks/claim-store.ts

```diff
@@ -90,6 +91,13 @@ export const sampleClaimObj = {
   }
 }
 
+export const sampleClaimDataWithEvidence = {
+  claim: {
+    ...sampleClaimObj.claim,
+    evidence: { rows: [{ type: EvidenceType.PHOTO.value, description: 'my photo evidence' }] }
```

Comment:
Is it worth doing this? i would just inline this into the route, that's what we would normally do for adding a field